xml_inscriptions/transcribed/NY.NY.NYU.L.X.143.xml

```diff
@@ -0,0 +1,183 @@
+                        <decoDesc>
+                            <decoNote>
+                                <p>animal (ram, cow, goat?) facing right</p>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                        </summary>
+
+                        <origin>
+                            <date evidence="#archaeology" notBefore="0001" notAfter="0300">first to
+                                third century CE</date>
+                            <placeName ref="unknown"/>
+                        </origin>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Bonfante_Fowlkes_2006"/>
+                        <biblScope unit="page">229-230, no. 84</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2023-11-14" who="Advaith Rege">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/>
+                <persName>
+                    <name type="praenomen"><expan><abbr>L</abbr><ex cert="low">ucius</ex></expan></name>
+                    <name type="gentilicium"><expan><abbr>Myn</abbr><ex cert="low">atius</ex></expan></name>
+                    <name type="cognomen">Phile</name>
```

Comment:
I wonder if we should understand this to be an abbreviated cognomen (for Philemon or something like that)? LGPN only seems to have Phile as a feminine name, so it strikes me as a bit odd here.